DDECal/ScreenConstraint.cc

```diff
@@ -157,32 +157,56 @@ void ScreenConstraint::CalculatePiercepoints(){
   void  ScreenConstraint::getPPValue(std::vector<std::vector<MultiDirSolver::DComplex> >& solutions,size_t solutionIndex,size_t dirIndex,double &avgTEC,double &error) const {
   if (itsAVGMode=="simple"){
     avgTEC=0;
+    error=1.;
+    uint nrch(0);
+    for(size_t ch=0;ch<_nChannelBlocks; ++ch){
+      if(isfinite(solutions[ch][dirIndex])){
+	double refphase=std::arg(solutions[ch][dirIndex]);
+	//TODO: more advance frequency averaging...
+	if (isfinite(solutions[ch][solutionIndex])){
+	avgTEC += std::arg(solutions[ch][solutionIndex]*std::polar<double>(1.0,-1*refphase))*itsFrequencies[ch]*phtoTEC;
+	nrch++;
+	  }
+      }
+    }
+    if (nrch>0)
+      avgTEC/=nrch;
+    /*
+    double mydelay=0;
```

Comment:
Why is this code commented out?

Reply:
It's new code, so I'm just accepting it as is for now